.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,14 +6,16 @@
 
 **What this PR does / why we need it**:
 
-**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
+**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
+Fixes #
 
 **Special notes for your reviewer**:
 
 **Release note**:
-<!--  Steps to write your release note:
-1. Use the release-note-* labels to set the release note state (if you have access)
-2. Enter your extended release note in the below block; leaving it blank means using the PR title as the release note. If no release note is required, just write `NONE`.
+<!--  Write your release note:
+1. Enter your extended release note in the below block. If the PR requires additional action from users switching to the new release, include the string "action required" (case insensitive, don't include quotation marks).
+2. If no release note is required, just write "NONE" (case insensitive, don't include quotation marks).
```

Comment:
I don't think its necessary to mention the case insensitivity or the quotation marks now. Once kubernetes/test-infra#5116 merges any way the string is written will be accepted.

Reply:
Updated.